go/terminalescaper/escaper.go

```diff
@@ -20,14 +20,20 @@ var vt100EscapeCodes = []EscapeCode{
 	EscapeCode{keyEscape, '[', '3', '5', 'm'},
 	EscapeCode{keyEscape, '[', '3', '6', 'm'},
 	EscapeCode{keyEscape, '[', '3', '7', 'm'},
+	EscapeCode{keyEscape, '[', '9', '0', 'm'},
+	// Reset foreground color
+	EscapeCode{keyEscape, '[', '3', '9', 'm'},
 	// Bold
 	EscapeCode{keyEscape, '[', '1', 'm'},
 	// Italic
 	EscapeCode{keyEscape, '[', '3', 'm'},
 	// Underline
 	EscapeCode{keyEscape, '[', '4', 'm'},
-	// Reset bold
+	// Reset bold (or doubly underline according to ECMA-48; fallback is code [22m)
+	// See https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters
 	EscapeCode{keyEscape, '[', '2', '1', 'm'},
+	// Normal intensity
+	EscapeCode{keyEscape, '[', '2', '2', 'm'},
```

Comment:
our `client/colors.go` uses `[22m` instead of `[21m` to reset bold, so added this.